### ISSUE TITLE
(CONFIG_FILE_IN_URL) update simple update

### ISSUE THREAD

**Issue by MrZimp** [developeranaz/URL-Magnet-Cloud-Uploader-Heroku#21]
Edit your readme and recommend users to make there config raw links like

https://gist.githubusercontent.com/[username]/23d5df10e67eb9d8157893e6df7d59ad/raw/[filename]

this will make it so if they just edit the gist it will update in the bot without needing to redeploy or change the link again so they can remove and add accounts easily

**Comment by developeranaz** [comment]
Thanks for this valuable knowledge, I searched for this feature on the web but it didn't work, now I got it, the gist needs little time to process its editing I will add it in readme soon , thanks😊👍

**Comment by MrZimp** [comment]
yes it's a super easy was to allow people to add and remove accounts without needing to redeploy, I learn that trick from other bots and I show your missing that, it's super useful when your working with mega accounts adding and removing is made easier, Thank you for making this code, it's super helpful and will be even more soon with gdrive TD been changes
